eng/code-quality-reports/src/main/java/com/azure/tools/checkstyle/checks/GoodLoggingCheck.java

```diff
@@ -117,9 +116,6 @@ public void visitToken(DetailAST ast) {
                     log(ast, String.format(NOT_CLIENT_LOGGER_ERROR, "Java System", CLIENT_LOGGER_PATH, methodCallName));
                 }
                 break;
-            case TokenTypes.METHOD_DEF:
-                checkForInvalidStaticLoggerUsage(ast);
```

Comment:
should we enforce static usage instead (after everything is converted)?

Reply:
Yes, I think it would be good to enforce the inverse of what this was previously checking after a while.